lib/ansible/module_utils/_internal/_no_six.py

```diff
@@ -0,0 +1,25 @@
+from __future__ import annotations
+
+import inspect
+
+from ansible.module_utils.common import warnings
+
+
+def deprecate(importable_name, *args):
```

Comment:
New functions should include type annotations:

```suggestion
def deprecate(importable_name: str, *args) -> object:
```